hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -69,6 +70,12 @@ public ClientInvocationServiceSupport(HazelcastClientInstanceImpl client) {
     }
 
     @Override
+    public <T> ICompletableFuture<T> invokeOnTarget(ClientRequest request, Address target) throws Exception {
+        final ClientInvocationFuture invoke = new ClientInvocation(client, request, target).invoke();
```

Comment:
Why don't you return the result of the invoke directly instead of storing it in a local variable? It doesn't make code more readable.